blocks/library/image/editor.scss

```diff
@@ -29,6 +29,7 @@
 
 	.wp-block-image.is-focused & {
 		display: block;
+		z-index: z-index( '.wp-block-image__resize-handlers' );
```

Comment:
We used to follow the CSS selector to name the z-index but granted it's too complex here.